cypress/integration/sortable_page_spec.ts

```diff
@@ -2,17 +2,34 @@ import { SortablePo } from '../support/sortable.po';
 
 describe('Sortable demo page test suite', () => {
   const sortable = new SortablePo();
-
   beforeEach(() => sortable.navigateTo());
 
   describe('Basic', () => {
     const basic = sortable.exampleDemosArr.basic;
 
     it('sortable items are placed at two sortable-wrappers', () => {
-      cy.get(`${ basic } ${ sortable.classWrapper }`).as('wrapper').eq(0)
+      cy.get(`${ basic } ${ sortable.classWrapper } `).as('wrapper').eq(0)
```

Comment:
1. Need to move basic testing scope to integration/sortable_page_spec.ts and full scope of tests to full/sortable_page_spec.ts
2. Need to move all logic to PO methods and reuse it in tests
3. Need to cover each DEMO, according to rule: 1 use-case = 1 describe, 1 step = 1 it
4. Need to make full coverage for all demos inside component